src/redux/books/books.js

```diff
@@ -0,0 +1,36 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (id) => ({
+  type: REMOVE_BOOK,
+  id,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        {
+          ...action.payload,
+          progress: {
+            currentChapter: 'NEW CHAPTER"',
+            completed: '0',
+          },
+        },
+      ];
```

Comment:
- [ ] I can see that you added a book action but you are required to implement adding book feature currently I can't see even a form please work on that

Reply:
alright, I fixed it